2016-03-28  Aurelien Jarno  <aurelien@aurel32.net>

	* sysdeps/unix/sysv/linux/sys/personality.h (UNAME26, FDPIC_FUNCPTRS,
	PER_LINUX_FDPIC): Add.

--- a/sysdeps/unix/sysv/linux/sys/personality.h
+++ b/sysdeps/unix/sysv/linux/sys/personality.h
@@ -26,7 +26,9 @@
    These occupy the top three bytes.  */
 enum
   {
+    UNAME26 = 0x0020000,
     ADDR_NO_RANDOMIZE = 0x0040000,
+    FDPIC_FUNCPTRS = 0x0080000,
     MMAP_PAGE_ZERO = 0x0100000,
     ADDR_COMPAT_LAYOUT = 0x0200000,
     READ_IMPLIES_EXEC = 0x0400000,
@@ -45,6 +47,7 @@ enum
   {
     PER_LINUX = 0x0000,
     PER_LINUX_32BIT = 0x0000 | ADDR_LIMIT_32BIT,
+    PER_LINUX_FDPIC = 0x0000 | FDPIC_FUNCPTRS,
     PER_SVR4 = 0x0001 | STICKY_TIMEOUTS | MMAP_PAGE_ZERO,
     PER_SVR3 = 0x0002 | STICKY_TIMEOUTS | SHORT_INODE,
     PER_SCOSVR3 = 0x0003 | STICKY_TIMEOUTS | WHOLE_SECONDS | SHORT_INODE,
